css/style.css

```diff
@@ -1,6 +1,180 @@
+* {
+  box-sizing: border-box;
+}
+
 html,
 body {
-  font-family: montserrat-font, sans-serif;
+  font-family: 'montserrat-font', sans-serif;
   font-size: 15px;
   font-weight: normal;
+  color: #000;
+}
+
+a {
+  text-decoration: none;
+}
+
+.btn {
+  padding: 10px 45px;
+  font-size: 13px;
+  line-height: 16px;
+  font-weight: bold;
+  text-transform: uppercase;
+  background-color: #fff;
+  border: 2px solid #e51937;
+  cursor: pointer;
+  transition: 1s;
+}
+
+.btn:hover {
+  transform: scale(1.1);
+}
+
+.header {
+  margin-bottom: 45px;
+}
+
+.container {
+  width: 1440px;
+}
+
+.container-header {
+  display: flex;
+  align-items: center;
+  padding-top: 17px;
+  padding-left: 31px;
+}
+
+.menu {
+  margin-left: auto;
+  margin-right: 68px;
+  margin-bottom: 5px;
+}
+
+.menu__link {
+  position: relative;
+  margin-right: 30px;
+  line-height: 18px;
+  color: #000000;
```

Comment:
Codacy found an issue: [Expected "#000000" to be "#000" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/GurevAleksey_itogovaya/pullRequest?prid=6093424)